src/styles/containers.ts

```diff
@@ -0,0 +1,14 @@
+export enum SIZES {
+  SM = 'sm',
+  MD = 'md',
+  LG = 'lg',
+}
+
+export const FONT_SIZES = {
```

Comment:
seems weird, why do you need FONT_SIZES as constant? better create a separate file as typography.ts and include common text styles